### PR TITLE
Add default value of EnableGeoData to chyt config

### PR DESCRIPTION
**Pull request by thenno**

### CODE REVIEW

File: yt/chyt/controller/internal/chyt/controller.go [hunk 1]
```diff
@@ -32,11 +32,13 @@ type Config struct {
 	AddressResolver           map[string]any       `yson:"address_resolver"`
 	EnableYandexSpecificLinks *bool                `yson:"enable_yandex_specific_links"`
 	ExportSystemLogTables     *bool                `yson:"export_system_log_tables"`
+	DefaultEnableGeoData      *bool                `yson:"default_enable_geo_data"`
 }
 
 const (
 	DefaultEnableYandexSpecificLinks = false
 	DefaultExportSystemLogTables     = false
+	DefaultEnableGeoData             = true
```
Comment (DimasKovas) [inline yt/chyt/controller/internal/chyt/controller.go:41]: Let's set it to `false` by default as we discussed in the chat

File: yt/chyt/controller/internal/chyt/controller.go [hunk 2]
```diff
@@ -32,11 +32,13 @@ type Config struct {
 	AddressResolver           map[string]any       `yson:"address_resolver"`
 	EnableYandexSpecificLinks *bool                `yson:"enable_yandex_specific_links"`
 	ExportSystemLogTables     *bool                `yson:"export_system_log_tables"`
+	DefaultEnableGeoData      *bool                `yson:"default_enable_geo_data"`
```
Comment (DimasKovas) [inline yt/chyt/controller/internal/chyt/controller.go:35]: The naming is inconsistent here.

It should be either just `EnableGeoData` (like `ExportSystemLogTables` option above), or we need to change the naming of the `ExportSystemLogTables` as well

File: yt/chyt/controller/internal/chyt/controller.go [hunk 3]
```diff
@@ -60,6 +62,13 @@ func (c *Config) ExportSystemLogTablesOrDefault() bool {
 	return DefaultExportSystemLogTables
 }
 
+func (c *Config) EnableGeoDataOrDefault() bool {
+	if c.DefaultEnableGeoData != nil {
```
Comment (DimasKovas) [inline yt/chyt/controller/internal/chyt/controller.go:66]: The naming of those `OrDefault` functions is `<FieldName>OrDefault()`

**Review by DimasKovas — CHANGES_REQUESTED** [review]

File: yt/chyt/controller/internal/chyt/controller.go (hunk 2 above)
Comment (thenno) [reply yt/chyt/controller/internal/chyt/controller.go]: Oh, I got the idea. Fixed.

**Review by DimasKovas — APPROVED** [review]

**Pull request closed without merging**